### DISCUSSION TITLE
vercel stop trigger deployment after pushing to GitHub.

### DISCUSSION THREAD

**Discussion by 1xo2** [vercel/next.js#45128] [Help]
Hello all,
vercel stop trigger deployment after pushing to GitHub.
i have tried to reconnect the app with GitHub and push again, but unsuccessfully.
also, when selecting redeploy from Vercel Deployments, it refers to the same commit. 

regards,
yaniv

**Comment by kfirfitousi** [comment]
One possible cause for this is that Vercel deduplicates deployments — which means that pushes that aren't affecting source files or environment variables are skipped.  
For example, changes to `README.md` won't trigger a new deployment.  
Source: https://vercel.com/guides/git-push-not-triggering-deploy

**Comment by 1xo2** [comment]
thanks for replying.. 
previously have tried to commit vercel.json...
but now tried to commit some content: with no effect

**Reply by kfirfitousi** [reply]
Changes to `vercel.json` require an authorization before deployment. Check the commit comments, there should be a link to authorize the deployment.
More info: https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel#public-repositories

**Reply by 1xo2** [reply]
thanks for participating.
at the end, vercel rejected the changes in vercel.json file.... so removing that file and all good...
